content/news/2021-01-24-internet-censorship.md

```diff
@@ -0,0 +1,123 @@
+The audacity of this government!
+
+As the years go by, our President, H.E Yoweri Kaguta Museveni, has been feeling a lot of pressure about the presidential seat.
+If elections were conducted freely and fairly, you would notice a declining trend in the number of votes his political party (NRM) has been getting over the years.
+Ugandans are tired, and they want change. They have been more vocal about the injustices that have been happening in the country and it has cost them their lives, even their families’.
+This resistance from the public has been causing anxiety in State House, and it’s starting to show.
+
+In the 2016 presidential elections, Ugandan authorities blocked access to Facebook, Twitter and WhatsApp saying the platforms would be used by the opposition to mobilize protests.
+(See [[this article from the BBC](https://www.bbc.com/news/world-africa-35601220))
+President Yoweri Museveni was facing a tough challenge from veteran activist [Kizza Besigye](https://twitter.com/kizzabesigye1?s=20), and he believed shutting down social media would help to curb the “threat”.
+Authorities also suspended mobile transfers of money.
+
+In 2019, the government introduced a tax on the use of the social media, which activists have called an attempt at controlling free speech.
+He said that the youth only use social media to gossip and spread wrong rumors (about him).
+(See [this article from the BBC](https://www.bbc.com/news/world-africa-44315675)) .
+He imposed OTT – Over the Top Tax and since then, it’s never been enough to simply have a good internet connection.
+You have to pay a daily fee of UGX 200 to your Internet Service Provider, or if you’re being rebellious, a VPN application installed on your phone or computer.
+He insisted that the revenue collected from the tax would help the country cope with the consequences of olugambo [gossiping].
+I don’t know.. but to me, these look like the actions of an increasingly anxious person in power.
+
+On 12th January 2021, the Uganda Communications Commission (UCC) ordered internet service providers in the country to immediately suspend any access and use of all messaging apps and social media platforms until further notice.
+This was done so suddenly and without prior notice that it took most of us a couple of failed OTT subscriptions to realize that social media platforms had indeed been blocked.
+Before that, internet speeds had become excruciatingly slow, it was any wonder we were able to get anything done in those few days.
+A few days after the social media block, the government decided to shut down the entire internet! (The social media ban was simply not cutting it)
+So when Google stopped working and our parents were restarting their phones several times, is when we realized that truly, there was no end to the surprises.
+
+During this [not so brief] period we were in total blackout. No internet services were available anywhere.
```

Comment:
use () parentheses for asides, not [] brackets.

Reply:
Alright